api/swagger.yaml

```diff
@@ -8593,6 +8593,7 @@ paths:
             - `label=<engine label>`
             - `membership=`(`accepted`|`pending`)`
             - `name=<node name>`
+            - `node-label=<node.label>`
```

Comment:
hm, looks like you changed the example value, instead of the "key"; we could discuss quickly, but I think the idea in https://github.com/moby/moby/issues/28209 was to have `--filter node.label=foobar` (so `node.label` instead of `node-label`)

@vdemeester ^^

Reply:
Yeah, to have `node.label`, `engine.label` and `label`. The dot `.` was maintly to be consistent with constraint if I rembember correctly :angel:

Reply:
Ah; yes; that was the reason